andela/travel_tool_front
# #159392486 Implement requests table(UI only)

#### What does this PR do?
Impement the table for requests

#### Description of Task to be completed?
- Create table for requests
- Create pagination component
- Style created components
- Test all created components
- Create route for requests page
- Set up sinon and enzyme for tests
- Link line-awesome cdn for icons

#### How should this be manually tested?
- Clone the repo
- Run ```cd travel-tool``` to enter the directory
- Run ```./docker.sh development``` to set up and start the app
- Navigate to `localhost:3000/requests` in the browser and you should see the table containing requests.

#### Any background context you want to provide?
This feature will have to be integrated with other components in the requests page

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[159392486](https://www.pivotaltracker.com/story/show/159392486)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-08-07 at 9 32 22 pm" src="https://user-images.githubusercontent.com/25195857/43800887-75c69cc8-9a89-11e8-90a4-eaf0e4a2bb29.png">
<img width="1440" alt="screen shot 2018-08-07 at 9 31 42 pm" src="https://user-images.githubusercontent.com/25195857/43800929-97452f5e-9a89-11e8-808f-53535a95ee72.png">

#### Questions:
N/A
